packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -96,34 +99,30 @@ export default function CommentActionButtons({
   };
 
   return (
-    <div className="flex items-center">
-      <QuaternaryButton
+    <div className="flex flex-row gap-3 items-center">
```

Comment:
Note to us, I'm not sure gap property is supported for flex items on Safari. You will have to check it manually.
Please confirm once you check.

Reply:
Ah, yes, it is not supported in Safari. I'll change this one out.